aksel.nav.no/website/package.json

```diff
@@ -30,7 +26,14 @@
     "playwright": "playwright test",
     "backup": "yarn tsx --env-file-if-exists=.env scripts/backup.ts",
     "generate-testpaths": "yarn tsx --env-file-if-exists=.env ./scripts/generate-testpaths.ts",
-    "watch:tw": "yarn tailwindcss -i ./components/styles/index.css -o ./dist/tw.css --watch"
+    "watch:tw": "yarn tailwindcss -i ./components/styles/index.css -o ./dist/tw.css --watch",
+    "update:sanity-schema": "cd ./sanity && yarn sanity schema extract --workspace default && yarn rename:sanity-schema && yarn sanity typegen generate",
+    "###⬆️ public (devs probably want these)": "###⬇️ private (ignore these)",
```

Comment:
Not that much of a fan of "dividers as code" like here 🤔

Reply:
Me neither 😓, i'll 🔥